ExpenseTracker/ExpenseTracker.Application/Mappings/TransferMappings.cs

```diff
@@ -18,40 +20,56 @@ public static TransferViewModel ToViewModel(this Transfer transfer)
         };
     }
 
-    public static UpdateTransferViewModel ToUpdateViewModel(this Transfer transfer)
+
+
+    public static Transfer ToEntity(this CreateTransferRequest transfer)
     {
-        return new UpdateTransferViewModel
+        return new Transfer
         {
-            Id = transfer.Id,
             Note = transfer.Note,
             Amount = transfer.Amount,
             Date = transfer.Date,
-            CategoryId = transfer.Category.Id
+            CategoryId = transfer.CategoryId,
+            Category = null,
+
         };
     }
-
-    public static Transfer ToEntity(this CreateTransferViewModel transfer)
+    public static Transfer ToEntity(this UpdateTransferRequest request)
     {
         return new Transfer
         {
-            Note = transfer.Note,
-            Amount = transfer.Amount,
-            Date = transfer.Date,
-            CategoryId = transfer.CategoryId,
+            Note = request.Note,
+            Amount = request.Amount,
+            Date = request.Date,
+            CategoryId = request.CategoryId,
             Category = null,
+            UserId = request.UserId
         };
     }
 
-    public static Transfer ToEntity(this UpdateTransferViewModel transfer)
+    public static TransferRequest ToTransferRequest(this UpdateTransferRequest request)
     {
-        return new Transfer
+        return new TransferRequest
         {
-            Id = transfer.Id,
-            Note = transfer.Note,
-            Amount = transfer.Amount,
-            Date = transfer.Date,
-            CategoryId = transfer.CategoryId,
-            Category = null,
+            TransferId = request.TransferId,
+            UserId = request.UserId,
+        };
+    }
+
+    public static GetCategoriesRequest ToGetCategoriesRequest(this GetTransfersRequest request)
+    {
+        return new GetCategoriesRequest
+        {
+            Search = request.Search,
+            UserId = request.UserId,
+        };
+    }
+    public static CategoryRequest ToCategoryRequest(this GetTransfersRequest request)
```

Comment:
there should be a space between these two lines

Reply:
Thanks